File: frontend/henn-laszlo/projects/content-data-access/src/lib/models/admin-article-details.ts
export interface AdminArticleDetails {
  readonly id: string;
  readonly slug: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly summaryHu: string | null;
  readonly summaryEn: string | null;
  readonly contentHu: string;
  readonly contentEn: string | null;
  readonly isPublished: boolean;
  readonly displayOrder: number;
  readonly createdAtUtc: string;
  readonly updatedAtUtc: string;
}

File: backend/src/Modules/Modules.Content.Application/Articles/GetAdminById/AdminArticleDetails.cs
namespace Modules.Content.Application.Articles.GetAdminById;

public sealed record AdminArticleDetails(
    Guid Id,
    string Slug,
    string TitleHu,
    string? TitleEn,
    string? SummaryHu,
    string? SummaryEn,
    string ContentHu,
    string? ContentEn,
    bool IsPublished,
    int DisplayOrder,
    DateTime CreatedAtUtc,
    DateTime UpdatedAtUtc);